packages/protocol/test-sol/utils/ECDSAHelper.sol

```diff
@@ -13,7 +13,7 @@ contract ECDSAHelper is Test {
     bytes32 _s
   ) public returns (bytes memory) {
     address SECP256K1Address = actor("SECP256K1Address");
-    deployCodeTo("out/SECP256K1.sol/SECP256K1.0.5.17.json", SECP256K1Address);
+    deployCodeTo("SECP256K1.sol:SECP256K1", SECP256K1Address);
```

Comment:
We already had to update this part twice because of changes in nightly builds of Foundry. Personally I would prefer to leave there json since Foundry doesn't try to resolve it but rather just uses it's path.

Reply:
Even with the path it was not working on my local.

This allows for usage with both 0.5 & 0.8, as well as being the same format that works for CI and local.